gdnative-derive/src/lib.rs

```diff
@@ -164,6 +197,44 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// - `no_editor`
 ///
 /// Hides the property from the editor. Does not prevent it from being sent over network or saved in storage.
+///
+/// ### `#[methods]`
+/// Adds the necessary information to an `impl` block for 
```

Comment:
This seems to be incomplete.
Do we want to link to the `#[methods]` documentation?

Then `#[methods]` would be documented in two places, but I think it's OK if we do it like this:
* here (`NativeClass`) we document how it's used, with all the sub-attributes
* in the `methods` doc, we document what it does technically (macro expansion).

So more or less how it's already in this PR. What do you think?

Reply:
That sounds good. I didn't know that we could use doc links, so I'll go ahead and put the doc link to the methods attribute to help direct them. I will also finish this up. It looks like I got distracted in the middle of it.